cred-manager/packages/@types/sourcecred/index.d.ts

```diff
@@ -924,7 +924,8 @@ declare module 'sourcecred' {
   export class CredGrainView {
     constructor(graph: CredGraph, ledger: Ledger);
     participants: () => ParticipantCredGrain[];
-
+    intervals: () => Interval[];
```

Comment:
This file is a generated type that runs on the sourcecred package, we shouldn't change these interfaces.

Exporting a new one from it like below is ok, but still think we should do that on a different file as this one will be overwritten again if we find a tool that generates better types for example